data/settings/icon-grid-C.json.in

```diff
@@ -36,29 +39,8 @@
     "net.sourceforge.Extremetuxracer.desktop",
     "net.sourceforge.Warmux.desktop"
   ],
-  "eos-folder-work.directory" : [
-    "libreoffice-calc.desktop",
-    "libreoffice-impress.desktop",
-    "gnome-calculator.desktop",
```

Comment:
Confirming that we don't want these apps in any default folder anymore: calculator, finance, resume, translation?